elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/util/impl/ElasticsearchDateHelper.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.elasticsearch.util.impl;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+
+import javax.xml.bind.DatatypeConverter;
+
+/**
+ * Various utilities to manipulate dates and comply with Elasticsearch date format.
+ *
+ * @author Guillaume Smet
+ */
+public final class ElasticsearchDateHelper {
+
+	private static final TimeZone ENCODING_TIME_ZONE = TimeZone.getTimeZone( "UTC" );
```

Comment:
In practice, there's no impact as UTC and GMT share the same current time.

Moreover, we only need to be consistent for a given backend and all the manipulations are done with the FieldBridges so we are on the safe side.